appveyor.yml

```diff
@@ -64,3 +64,4 @@ test_script:
  - echo %PATH%
  - '%DC% --version'
  - make -f win32.mak rdmd dustmite ddemangle changed
+ - make -f win32.mak test
```

Comment:
I should add `VERBOSE_RDMD_TEST=1` here so that we actually see the tests being performed.